SoftLayer/CLI/order/item_list.py

```diff
@@ -47,9 +55,22 @@ def cli(env, package_keyname, keyword, category):
     sorted_items = sort_items(items)
 
     categories = sorted_items.keys()
-    for catname in sorted(categories):
-        for item in sorted_items[catname]:
-            table.add_row([catname, item['keyName'], item['description'], get_price(item)])
+    if prices:
+        table = formatting.Table(COLUMNS_ITEM_PRICES, title="CRMax = CapacityRestrictionMaximum, "
```

Comment:
As with `create-options` combine the capacity columns in to a single column called `Restrictions` with the format `{Min}-{Max} {TYPE}`

Reply:
Done.